atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/config/TargetedSweepInstallConfig.java

```diff
@@ -28,17 +29,16 @@
 @Value.Immutable
 public class TargetedSweepInstallConfig {
     /**
-     * If true, information for targeted sweep will be persisted to the sweep queue. This is necessary to be able to
-     * run targeted sweep.
-     *
-     * Once you decide to use targeted sweep, DO NOT set this to false without consulting with the AtlasDB team. Doing
-     * so will cause targeted sweep to not be aware of any writes occurring while this is false, and you will have to
-     * use legacy sweep to sweep those writes. If you wish to pause targeted sweep, that can be done by setting the live
-     * reloadable {@link TargetedSweepRuntimeConfig#enabled()} parameter to false.
+     * @deprecated Disabling sweep queue writes is not supported anymore in order to maintain correctness while running
+     * transactions4.
+     * If you wish to pause targeted sweep, that can be done by setting the live reloadable
+     * {@link TargetedSweepRuntimeConfig#enabled()} parameter to true.
      */
+    @Deprecated
     @Value.Default
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
```

Comment:
Interesting: so this means we still read this from config, but don't write it when serializing (if I'm reading this correctly). Is this what we want?